api/src/main/java/org/motechproject/nms/api/web/BaseController.java

```diff
@@ -46,6 +46,7 @@
     public static final String NOT_FOUND = "<%s: Not Found>";
     public static final String NOT_AUTHORIZED = "<%s: Not Authorized>";
     public static final String NOT_DEPLOYED = "<%s: Not Deployed In State>";
+    public static final String INVALID_TYPE = "<%s: Invalid Type>";
```

Comment:
These are not return types. They are just to print error for debugging purposes internally. So, we don't need to reach out to IMI.